src/app/helium.csproj

```diff
@@ -1,26 +1,26 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>Helium</RootNamespace>
+    <AssemblyName>helium</AssemblyName>
     <AssemblyVersion>0.5.0.0</AssemblyVersion>
-  </PropertyGroup>
```

Comment:
I had to add <LangVersion>8.0</LangVersion> for this to build locally for me.  It's probably due to the configuration on my machine, but probably should either call it out here or in the prereqs? (Unless it's really just me :) )

Reply:
This was due to different versions of VS, so we added for consistency